src/getTypeFromValue.js

```diff
@@ -0,0 +1,4 @@
+import { GraphQLString } from 'graphql';
+
+// FIXME obviously very weak
+export default (name, value) => GraphQLString; // eslint-disable-line no-unused-vars
```

Comment:
I was wondering whether we should provide all existing values from the dataset. I don't see how we could distinguish int from float otherwise